File: src/main/java/com/sleepwalker/sleeplib/util/request/ConsoleRequestSender.java
package com.sleepwalker.sleeplib.util.request;

import com.sleepwalker.sleeplib.util.ServerUtil;
import net.minecraft.server.MinecraftServer;
import net.minecraft.util.Util;
import net.minecraft.util.text.ITextComponent;
import net.minecraftforge.fml.server.ServerLifecycleHooks;

import javax.annotation.Nonnull;
import java.util.UUID;

public class ConsoleRequestSender implements IRequestSender {

    @Override
    public boolean hasPermission(@Nonnull String node) {
        return true;
    }

    @Override
    public void sendMessage(@Nonnull ITextComponent message, boolean notifyAdmins) {
        MinecraftServer server = ServerLifecycleHooks.getCurrentServer();
        if(server != null){
            server.sendMessage(message, Util.DUMMY_UUID);
        }
        if(notifyAdmins){
            ServerUtil.broadcastToAdmins(this, message);
        }
    }

    @Nonnull
    @Override
    public String getName() {
        return "Console";
    }

    @Nonnull
    @Override
    public UUID getUUID() {
        return Util.DUMMY_UUID;
    }
}

File: src/main/kotlin/com/sleepwalker/sleeplib/util/request/ConsoleRequestSender.kt
package com.sleepwalker.sleeplib.util.request

import com.sleepwalker.sleeplib.util.broadcastToAdmins
import net.minecraft.util.Util
import net.minecraft.util.text.ITextComponent
import net.minecraftforge.fml.server.ServerLifecycleHooks
import java.util.*

object ConsoleRequestSender : IRequestSender {
    override val name: String
        get() = "Console"
    override val uuid: UUID
        get() = Util.DUMMY_UUID
    override fun hasPermission(node: String): Boolean = true
    override fun sendMessage(message: ITextComponent, notifyAdmins: Boolean) {
        ServerLifecycleHooks.getCurrentServer()?.sendMessage(message, Util.DUMMY_UUID)
        if (notifyAdmins) {
            broadcastToAdmins(this, message)
        }
    }
}